nats-base-client/headers.ts

```diff
@@ -81,8 +85,13 @@ export class MsgHdrsImpl implements MsgHdrs {
     const lines = s.split(MsgHdrsImpl.CRLF);
     const h = lines[0];
     if (h !== MsgHdrsImpl.HEADER) {
-      const str = h.replace(MsgHdrsImpl.HEADER, "");
-      mh.error = parseInt(str, 10);
+      let str = h.replace(MsgHdrsImpl.HEADER, "");
+      mh.code = parseInt(str, 10);
+      const scode = mh.status.toString();
+      mh.set("Status", scode);
+      str = str.replace(mh.status.toString(), "");
```

Comment:
would `mh.status.toString()` not be `scode`?

Reply:
the code above is completely broken - some undo killed the value.